heron/schedulers/tests/java/com/twitter/heron/scheduler/local/LocalSchedulerTest.java

```diff
@@ -74,18 +72,25 @@ public void testClose() throws Exception {
   public void testOnSchedule() throws Exception {
     Mockito.doNothing().
         when(scheduler).startExecutorMonitor(Mockito.anyInt(), Mockito.any(Process.class));
-    Process mockProcess = Mockito.mock(Process.class);
-    Mockito.doReturn(mockProcess).
-        when(scheduler).startExecutorProcess(Mockito.anyInt());
+    Process[] mockProcesses = new Process[3];
+    for (int i = 0; i < 3; i++) {
+      mockProcesses[i] = Mockito.mock(Process.class);
+      Mockito.doReturn(mockProcesses[i]).when(scheduler).startExecutorProcess(i);
+    }
 
     PackingPlan packingPlan = Mockito.mock(PackingPlan.class);
+    Set<PackingPlan.ContainerPlan> containers = new HashSet<>();
+    containers.add(PackingTestUtils.testContainerPlan(1));
+    containers.add(PackingTestUtils.testContainerPlan(2));
```

Comment:
We could use container id 3 here instead of 2 to assert that it's ok to not be consecutive.